force-app/main/default/reports/unfiled$public/Active_Orders_Awaiting_Delivery_NX8.report-meta.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Report xmlns="http://soap.sforce.com/2006/04/metadata">
+    <columns>
+        <field>ParcelLocker__c.Cells__r$Name</field>
+    </columns>
+    <columns>
+        <field>ParcelLocker__c.Cells__r.CellOrders__r$Name</field>
+    </columns>
+    <columns>
+        <field>ParcelLocker__c.Cells__r.CellOrders__r$State__c</field>
+    </columns>
+    <filter>
+        <criteriaItems>
+            <column>ParcelLocker__c.Cells__r.CellOrders__r$State__c</column>
+            <columnToColumn>false</columnToColumn>
+            <isUnlocked>true</isUnlocked>
+            <operator>equals</operator>
+            <value>Pending delivery,Delivering</value>
```

Comment:
It should be `Pending delivery` only.

Reply:
Report and Report Type were changed for displaying Cell Orders with "Pending Delivery" status.